src/main/java/frc/robot/RobotContainer.java

```diff
@@ -21,10 +25,15 @@
 public class RobotContainer {
   // The robot's subsystems and commands are defined here...
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
+  FloorIntakeSubsystem floorIntake = new FloorIntakeSubsystem();
```

Comment:
It's good to restrict the surface area of a class when you can. Each member of a class has a "visibility", and without specifying one explicitly, it will be "package private". This means that other classes in this package can access `floorIntake`.

You can make it private by putting `private` in front of it (see the line above). This is a "good" thing to do because it constrains the way other parts of the codebase can interact with this class which makes those interactions simpler to reason about.

Here's a good writeup on [access modifiers](https://www.baeldung.com/java-access-modifiers) if you're curious.

`final` is also a good modifier to have here, since it means that code won't be able to re-assign this variable. This will prevent us as authors of this code from mistakenly reassigning this instance variable in the future.

Reply:
done B)